### PR TITLE
Security page

### PR DESCRIPTION
**Pull request by isoos**
- #2618 
- not much content yet, but easier to extend once we have a start on it

### CODE REVIEW

File: app/lib/frontend/templates/views/security.mustache [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      The Dart team takes the security of Pub and its published packages seriously.
+      This page describes how to report any vulnerabilities you may find.
+  </p>
+
+  <h3>Reporting vulnerabilities</h3>
+  <p>
+      In the rare event that you find a vulnerability in Pub site itself, email us at
+      <a href="mailto:product-security@google.com">product-security@google.com</a>.
+      For more information about how Google handles security issues, see
+      <a href="https://www.google.com/about/appsecurity/">Google’s security philosophy</a>.
```
Comment (jonasfj) [inline app/lib/frontend/templates/views/security.mustache]: @mit-mit, do you have any comments on the contents here?

If we want `product-security@google.com` handling, we probably have to do some setup.

File: app/lib/frontend/templates/views/security.mustache [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{{! Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+    for details. All rights reserved. Use of this source code is governed by a
+    BSD-style license that can be found in the LICENSE file. }}
+<div style="max-width: 600px">
+  <h2>Security</h2>
+  <p>
+      The Dart team takes the security of Pub and its published packages seriously.
```
Comment (mit-mit) [inline app/lib/frontend/templates/views/security.mustache]: I think we should be more general here: s/of Pub and its published packages/of Pub.dev/

File: app/lib/frontend/templates/views/security.mustache (hunk 1 above)
Comment (mit-mit) [reply app/lib/frontend/templates/views/security.mustache]: I already did that (see cl/261665758)

File: app/lib/frontend/templates/views/security.mustache [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+      This page describes how to report any vulnerabilities you may find.
+  </p>
+
+  <h3>Reporting vulnerabilities</h3>
+  <p>
+      In the rare event that you find a vulnerability in Pub site itself, email us at
```
Comment (mit-mit) [inline app/lib/frontend/templates/views/security.mustache]: s/Pub site itself/Pub.dev/

**Pull request merged**